### DISCUSSION TITLE
Trunk based development

### DISCUSSION THREAD

**Discussion by sanju-naik** [MobileNativeFoundation/discussions#221] [General]
Hey all 👋 

I am starting this discussion to understand are companies following practices like trunk-based development i.e merging smaller PRs and merging them frequently to ship app updates every week ?

- Are you following trunk-based development? 
- If yes, then what are some of the strategies you have applied to ensure smaller code changes are being merged frequently?

**Comment by esoxjem** [comment]
- TBD using short-lived feature branches works well: https://trunkbaseddevelopment.com/styles/#short-lived-feature-branches
- Release trains for releasing at a fixed cadence

There is no one size fits all strategy. Your team's context and culture are important to pick the right strategy that would be sucessful for you.

https://www.runway.team/blog/mobile-releases-feature-based-or-release-train
